src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_RootScreen/Dashboard_RootScreen.tsx

```diff
@@ -128,19 +128,16 @@ export const Dashboard_RootScreen: React.FC<RootProps> = ({
       includeJourneyHistory: true,
     });
 
-  const setCurrentLocationOrRequest = useCallback(
-    async function setCurrentLocationOrRequest() {
-      if (currentLocation) {
+  const setCurrentLocationOrRequest = useCallback(async () => {
+    if (currentLocation) {
+      setCurrentLocationAsFrom();
+    } else {
+      const status = await requestLocationPermission();
+      if (status === 'granted') {
         setCurrentLocationAsFrom();
-      } else {
-        const status = await requestLocationPermission();
-        if (status === 'granted') {
-          setCurrentLocationAsFrom();
-        }
       }
-    },
-    [currentLocation, setCurrentLocationAsFrom, requestLocationPermission],
-  );
+    }
+  }, [currentLocation, setCurrentLocationAsFrom, requestLocationPermission]);
```

Comment:
Note: the only change here is `() => {` instead of `function setCurrentLocationOrRequest() {` since the name was repeated twice